### PR TITLE
Created a Ban list/Tested games

### PR DESCRIPTION
**Pull request by Tatohead**
Sorted alphabetically. Can be formatted better. I think it should be added to the info section since people may not want to read FAQ

### CODE REVIEW

File: info/GameTestList.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Some games may ban the players while the desktop goose is being played in the background.
+
+| Symbol | Meaning                        |
+| ------ |:------------------------------:|
+| ✔️    | Okay to play the goose          |
+| ❌    | Will get banned using the goose |
+
+## List of Tested Games
+
+* CS:GO ✔️
```
Comment (VukAnd) [inline info/GameTestList.md]: image
You could use a table for the games as well (obvious mockup with inspect element here so didn't change the last one, you get the point anyway)

Comment (Tatohead) [reply info/GameTestList.md]: Oh, I'll change it up then for better formatting.

**Comment by VukAnd** [comment]
image
Also, more information was added to the FAQ, you may want to include those as well.

**Comment by Tatohead** [comment]
I'll try to include more symbols to use based on that image you just sent

**Comment by Tatohead** [comment]
I think I wrote all of the ones listed so far.

**Comment by VukAnd** [comment]
The list was updated to include ROBLOX, would you mind adding that as well?

**Comment by VukAnd** [comment]
You should probably also add a link to the list in the README

**Comment by Tatohead** [comment]
Is the Roblox item on that list not showing for yours? Wait I think you meant the Runescape 2 one I'll update it again.

**Pull request merged**